### PR TITLE
Declared license mapping update

### PR DESCRIPTION
**Pull request by tsteenbe**
See individual commits for details

### CODE REVIEW

File: spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt [hunk 1]
```diff
@@ -86,6 +86,7 @@ object SpdxDeclaredLicenseMapping {
         """Apache-2.0 */ &#39; &quot; &#x3D;end --
 
  """.trimIndent() to APACHE_2_0.toExpression(),
+        "Apache-style" to APACHE_2_0.toExpression(),
```
Comment (sschuberth) [inline spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt]: Please use a permalink that cannot break in the commit message:

https://github.com/saksdirect/gfc-collection/blob/d65da56/build.sbt#L42

File: spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt [hunk 2]
```diff
@@ -127,6 +127,7 @@ object SpdxDeclaredLicenseMapping {
         "BSD style license" to BSD_3_CLAUSE.toExpression(),
         "BSD*" to BSD_3_CLAUSE.toExpression(),
         "BSD-like license" to BSD_3_CLAUSE.toExpression(),
+        "BSD-Style" to BSD_3_CLAUSE_ATTRIBUTION.toExpression(),
```
Comment (sschuberth) [inline spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt]: Please use a permalink that cannot break in the commit message:

https://github.com/jsuereth/scala-arm/blob/366e8d9/build.sbt#L68

Also, I guess this should make to plain `BSD_3_CLAUSE` (not attribution).

File: spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt (hunk 1 above)
Comment (tsteenbe) [reply spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt]: Fixed

File: spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt (hunk 2 above)
Comment (tsteenbe) [reply spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt]: Fixed

**Comment by tsteenbe** [comment]
CI is flacky and this is a minor change so merging without completing the CI builds.

**Pull request merged**